build/document-extractor.js

```diff
@@ -43,8 +43,9 @@ function extractSections($) {
   const iterable = [...$("#_body")[0].childNodes];
 
   let c = 0;
+  const tagNames = new Set(["h2", "h3", "h4"]);
   iterable.forEach((child) => {
-    if (child.tagName === "h2" || child.tagName === "h3") {
+    if (tagNames.has(child.tagName)) {
```

Comment:
Since we're doing this comparison quite a lot, can we use a RegExp instead for performance?

```suggestion
    if (/^h[2-4]$/.test(child.tagName)) {
```

Reply:
Yeah, it will be quite a bit faster in terms of time complexity/performance to use `RegExp.test(string)` for string matching. The original chained `||` logic isn't bad but RegExp.prototype.test() will be the fastest. Nice one!